src/cascadia/TerminalApp/AppLogic.h

```diff
@@ -101,7 +101,7 @@ namespace winrt::TerminalApp::implementation
         winrt::com_ptr<LanguageProfileNotifier> _languageProfileNotifier;
         wil::unique_folder_change_reader_nothrow _reader;
 
-        TerminalApp::ContentManager _contentManager{ *winrt::make_self<implementation::ContentManager>() };
+        TerminalApp::ContentManager _contentManager{ winrt::make<implementation::ContentManager>() };

```

Comment:
Want to know something even more fun? Since these are in the same module, if you just do `TerminalApp::ContentManager _contentManager;` it will use a fast path to call `winrt::make<...>` directly. You can treat the projected type as "law"! I forgot about this until I saw it with `make` in.

You don't need to make this change, I just think it's v.cool. Check it out:

# In `Foo.g.cpp`

```c++
// WARNING: Please don't edit this file. It was generated by C++/WinRT v2.0.210825.3

void* winrt_make_TerminalApp_Foo()
{
    return winrt::detach_abi(winrt::make<winrt::TerminalApp::factory_implementation::Foo>());
}
WINRT_EXPORT namespace winrt::TerminalApp
{
    Foo::Foo() :
        Foo(make<TerminalApp::implementation::Foo>())
    {
    }
}
```

It provides a linkable constructor for `TerminalApp::Foo`, and doesn't even generate the one that calls `RoActivateInstance` in `winrt/whatever.h`